clients/privacy-center/README.md

```diff
@@ -0,0 +1,53 @@
+# FidesOps Privacy Center
+
+Privacy center for FidesOps. A web application built in Next.js with the FidesUI
+component library.
+
+## Configuration
+
+You may configure the appearance of this web application at build time by modifying the `config.json` file inside the `config` directory. You're able to control:
+
+- The header of the document
+- The descriptive text of the document
+- Which actions are present
+- The titles and descriptions of, and personally identifying information required by, each action
+
+You can also add any CSS you'd like to the page by adding it to the `config.css`
+file inside the `config` directory.
+
+Additionally, if you're handy with React and are feeling a little brave you can
+customize the entire application by modifying the TypeScript source code in
+`pages/index.tsx` and `components/RequestModal.tsx`.
+
+## Development
+
+To serve this application locally, run:
+
+```bash
+npm run dev
```

Comment:
I believe this assumes that `next` is already installed, we should add explicit instructions to `npm install next` if it's not already installed

Reply:
It doesn't need to be installed globally – it's [a local dependency](https://github.com/elliotbonneville/fidesops/blob/main/clients/privacy-center/package.json#L24), and NPM scripts can pick up on that.

Reply:
FWIW I'm only making that comment as when I ran `npm run dev` for the first time on this fork it complained `next` couldn't be found. I'm not sure the local deps are auto-installing in that case?

Reply:
Yep, I've added that command to the readme.